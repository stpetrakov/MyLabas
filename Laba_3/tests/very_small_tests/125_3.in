32448 25995 13464 14615 8921 23362 32426 25983 22792 29891 13122 9381 25610 23714 14899 18530 29282 23337 32564 32735 24491 25173 24025 25522 8236 6946 2498 3401 2733 22536 18195 15756 29833 4294 1827 22206 21509 19294 15737 11684 23776 5729 244 30186 19243 30828 19785 19906 20571 30963 30899 11470 22173 9789 12773 14912 3689 15536 6724 20032 31322 21327 20563 8846 27965 12673 32366 30312 11880 12820 16828 10670 14413 22401 32095 4932 17742 30066 28141 5816 7891 4993 4388 15652 25242 27971 13785 21846 12879 13189 18482 24367 349 29397 11797 17993 29955 17221 18400 12252 2730 9579 26971 24547 30838 12460 24299 31098 31771 30395 21256 7819 3058 16476 10224 12969 7756 1902 6557 17724 31494 21032 15349 26335 19293 