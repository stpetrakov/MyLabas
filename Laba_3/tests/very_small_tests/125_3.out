244 349 1827 1902 2498 2730 2733 3058 3401 3689 4294 4388 4932 4993 5729 5816 6557 6724 6946 7756 7819 7891 8236 8846 8921 9381 9579 9789 10224 10670 11470 11684 11797 11880 12252 12460 12673 12773 12820 12879 12969 13122 13189 13464 13785 14413 14615 14899 14912 15349 15536 15652 15737 15756 16476 16828 17221 17724 17742 17993 18195 18400 18482 18530 19243 19293 19294 19785 19906 20032 20563 20571 21032 21256 21327 21509 21846 22173 22206 22401 22536 22792 23337 23362 23714 23776 24025 24299 24367 24491 24547 25173 25242 25522 25610 25983 25995 26335 26971 27965 27971 28141 29282 29397 29833 29891 29955 30066 30186 30312 30395 30828 30838 30899 30963 31098 31322 31494 31771 32095 32366 32426 32448 32564 32735 